config/locales/de.yml

```diff
@@ -47,56 +47,56 @@ de:
         precision: 0
         format: '%u%n'
   homepage:
-    intro: "<strong>codebar</strong> is a charity that facilitates the growth of a diverse tech community by running free regular programming workshops for minority groups in tech."
+    intro: "<strong>codebar</strong> ist eine Wohltätigkeitsorganisation, die das Wachstum einer vielfältigen Tech-Community fördert, indem sie regelmäßig kostenlose Programmier-Workshops für Minderheiten in der Tech-Branche anbietet."
     explanation: "Unser Ziel ist unterrepräsentierten Menschen zu programmieren in einer sicheren und kollaborative Umgebung lernen und erweitern ihre Karrierechancen zu ermöglichen. Um dies zu erreichen betreiben wir kostenlose regelmäßige Workshops, regelmäßige Einzelveranstaltungen und versuchen, die Möglichkeiten für unsere Schüler die Technologie und Programmierung zugänglicher zu schaffen."
     participate:
-      students_title: "Students"
-      students_explanation_p1: "Our students come from a variety of backgrounds. Some want to become full-time developers, whereas some would like to learn the basics of coding in a supportive environment."
-      students_explanation_p2: "Sign up to join our vibrant community and attend our workshops."
-      students_button: "Sign up as a student"
+      students_title: "Studenten"
+      students_explanation_p1: "Unsere Studenten kommen aus den unterschiedlichsten Bereichen. Einige wollen Vollzeitentwickler werden, während andere die Grundlagen des Programmierens in einem unterstützenden Umfeld erlernen möchten."
+      students_explanation_p2: "Melden Sie sich an, um unserer lebendigen Gemeinschaft beizutreten und an unseren Workshops teilzunehmen."
+      students_button: "Als Student anmelden"
       coaches_title: "Coaches"
-      coaches_explanation_p1: "We encourage new students to work on our HTML/CSS, JavaScript, Ruby, Python or Git <a href='http://tutorials.codebar.io'>tutorials</a>. We also help them understand programming concepts."
-      coaches_explanation_p2: "Therefore we are always on the lookout for more developers to join our community and help coach at our workshops."
-      coaches_button: "Sign up as a coach"
+      coaches_explanation_p1: "Wir ermutigen neue Studenten, mit unseren HTML/CSS, JavaScript, Ruby, Python oder Git <a href='http://tutorials.codebar.io'>Tutorials</a> zu arbeiten. Wir helfen ihnen auch, Programmierkonzepte zu verstehen."
+      coaches_explanation_p2: "Deshalb sind wir immer auf der Suche nach weiteren Entwicklern, die sich unserer Gemeinschaft anschließen und uns bei unseren Workshops unterstützen."
+      coaches_button: "Als Coach anmelden"
```

Comment:
You could say `Coach` in German just fine, but it's a little blurry what it actually means. Maybe `Lehrer` which translates to `teacher`? Then again there could be a reason the English copy doesn't use `teacher`.

Reply:
I'd stick with `Coach` - we also say `Workshop` and `Community` in the german text. I'd say these words are established and well understood in german.